test/static-thread-pool.cc

```diff
@@ -151,3 +163,27 @@ TEST(StaticThreadPoolTest, Spawn) {
 
   *e();
 }
+
+TEST(StaticThreadPoolTest, Concurrent) {
+  std::vector<int> digits = {1, 2, 3};
+
+  auto id = std::this_thread::get_id();
+  auto e = [&]() {
+    return StaticThreadPool::Spawn(
+        "concurrent test",
+        Iterate(digits)
+            | ConcurrentOrdered([&]() {
+                auto* context = Scheduler::Context::Get();
+                return Map(Let([&](int& i) {
+                  EXPECT_NE(id, std::this_thread::get_id());
+
+                  // Should be executed on a cloned StaticThreadPool context.
+                  EXPECT_NE(context, Scheduler::Context::Get());
```

Comment:
```suggestion
                  EXPECT_NE(context, Scheduler::Context::Get());
                  EXPECT_EQ(context, Scheduler::Context::Get()->parent());
```

Reply:
Since we're going to remove the parent let's instead check that the current context's scheduler is the same as the outer context's scheduler!